truffle/contracts/Load/LoadContract.sol

```diff
@@ -213,46 +209,19 @@ contract LoadContract is Ownable {
         external
         notDeprecated
     {
-        createNewShipment2(_shipmentUuid, _fundingType, _contractedAmount, "", "", address(0x0));
-    }
-
-    /** @notice Associates a Vault URL with this Shipment.
-      * @param _shipmentUuid bytes16 Shipment's UUID.
-      * @param _vaultUri string URI of the external vault.
-      * @dev Emits VaultUri on success
-      */
-    function setVaultUri(bytes16 _shipmentUuid, string calldata _vaultUri)
-        external
-        shipmentExists(_shipmentUuid)
-    {
-        allShipmentData[_shipmentUuid].setVaultUri(_shipmentUuid, _vaultUri);
-    }
-
-    /** @notice Associates a Vault Hash with this Shipment.
-      * @param _shipmentUuid bytes16 Shipment's UUID.
-      * @param _vaultHash string Hash of the external vault.
-      * @dev Emits VaultHash on success.
-      */
-    function setVaultHash(bytes16 _shipmentUuid, string calldata _vaultHash)
-        external
-        shipmentExists(_shipmentUuid)
-    {
-        allShipmentData[_shipmentUuid].setVaultHash(_shipmentUuid, _vaultHash);
+        createNewShipment2(_shipmentUuid, _fundingType, _contractedAmount, address(0x0));
     }
 
     /** @notice Creates a new Shipment and stores it in the Load Registry.
       * @param _shipmentUuid bytes16 representation of the shipment's UUID.
       * @param _fundingType Escrow.FundingType Type of funding for the escrow.  Can be NO_FUNDING for no escrow.
       * @param _contractedAmount uint256 Escrow token/ether amount if escrow is
       defined.
-      * @param _vaultUri string The Uri of Vault
-      * @param _vaultHash string The hash of Vault
-      * @param _carrierAddress address The addres of the carrier for this shipment
+      * @param _carrierAddress address The address of the carrier for this shipment
       * @dev Emits ShipmentCreated on success.
       */
     function createNewShipment2(bytes16 _shipmentUuid, Escrow.FundingType
-    _fundingType, uint256 _contractedAmount, string memory _vaultUri, string
-    memory _vaultHash, address _carrierAddress)
+    _fundingType, uint256 _contractedAmount, address _carrierAddress)
```

Comment:
Sorry, I might be out of the loop here -- Why the `2` in the name? What's that signify? Can we get a comment explaining it?

Reply:
createNewShipment2 was created during the last round of contract changes I made before the new contract. At that time, the goal was to have another createNewShipment function that can take in vaultUrl, vaultHash and carrierAddress, set them inside the process of creating a new shipment, so that we can have a single function call that can create a new shipment. 
At the beginning, I tried to use function overloading, i.e. using the same name createNewShipment for this new function. But there were some problems when calling the overloaded functions from unit test and web3. Then, after discussion with Adam and Lucas, we decided to create another createNewShipment with another name while keeping the old one for backward compatibility.  We brainstormed several names for the new function, and decided to use createNewShipment2 because of the other candidates were kind of wordy. 

The last story is after the new contract added, we found out we do not want to call the setVaultUri and setVaultHash inside the LoadContract anymore. So, the only difference of createNewShipment to createNewShipment2 is that createNewShipment2 sets the carrier inside. 

The END...  : )
Please let us know if you had another name in mind or a better design. Thanks!